src/patternfly/demos/Card/card-content-2.hbs

```diff
@@ -0,0 +1,26 @@
+{{#> card-header}}
+  {{#> l-flex flex--modifier="pf-m-align-right"}}
+    {{#> l-flex flex--modifier="pf-m-grow"}}
+      {{#> brand brand--attribute='src="/assets/images/pf-logo-small.svg" alt="Logo"'}} 
+      {{/brand}}
+    {{/l-flex}}
+    {{#> dropdown id=(concat card--id "-dropdown-kebab-right-aligned") dropdown--IsActionMenu="true" dropdown-toggle--modifier="pf-m-plain" dropdown--HasKebabIcon="true" aria-label="Actions"}}
+    {{/dropdown}}
+    {{#> check}} 
+      {{#> check-input check-input--attribute=(concat 'id="' card--id '-check" name="' card--id '-check" aria-labelledby="' card--id '-check-label"')}}{{/check-input}}
+    {{/check}}
+  {{/l-flex}}
+{{/card-header}}
+{{#> card-body}}
+  {{#> content content--modifier="pf-m-no-margin"}}
+    <h4 id="{{concat card--id "-check-label"}}">ActiveMQ</h4>
+  {{/content}}
+  {{#> content}}
+    <small>Provided by Red Hat</small>
+  {{/content}}
+  {{#> content content--modifier="pf-m-no-margin"}}
+    <small>
```

Comment:
I think this is an OK use of the `<small>` element, too. But we want the text to be black. Not sure how to do that.